infer/src/checkers/liveness.ml

```diff
@@ -211,6 +211,13 @@ let get_captured_by_ref_invariant_map proc_desc =
   let cfg = ProcCfg.Exceptional.from_pdesc proc_desc in
   CapturedByRefAnalyzer.exec_cfg cfg () ~initial:VarSet.empty
 
+module IntLitSet = Caml.Set.Make (IntLit)
+
+let whitelisted_constants =
+  let int_lit_constants =
+    List.map ~f:(fun el -> IntLit.of_string el) Config.liveness_whitelist_constant
```

Comment:
If the user passes bad input we'll crash here. Maybe worth getting a nice error message? E.g.

```
fun el -> try IntLit.of_string el with Invalid_argument -> L.die UserError "Ill-formed option  '%s' for --liveness-whitelist-constant: an integer was expected"
```

Reply:
Good point. Thank you for the helpful feedback!